StudyplusSDK-V2.podspec

```diff
@@ -3,13 +3,12 @@ Pod::Spec.new do |s|
   s.version               = "2.0.1"
   s.summary               = "StudyplusSDK-V2 is Studyplus iOS SDK for Swift"
   s.homepage              = "https://info.studyplus.jp"
-  s.license               = { :type => "MIT", :file => "LICENSE" }
+  s.license               = "MIT"
   s.source                = { :git => "https://github.com/studyplus/Studyplus-iOS-SDK-V2.git", :tag => s.version }
   s.source_files          = "StudyplusSDK", "Lib/StudyplusSDK/**/*.{swift}"
-  s.requires_arc          = true
   s.platform              = :ios, '11.0'
   s.ios.deployment_target = '11.0'
   s.ios.frameworks        = ['UIKit', 'Foundation']
   s.author                = { 'Studyplus inc' => 'developer-all@studyplus.jp' }
-  s.swift_versions        = ['4.2', '5.0']
```

Comment:
Target 5.1 and above.

https://github.com/kean/Nuke/blob/9.3.1/Nuke.podspec#L15